app/models/bulk_download.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+require 'zip'
+
+##
+# This class is used to create a zip file
+# of all the files in a FileSet
+class BulkDownload
+  attr_accessor :zip_file_name, :files
+  ##
+  # @param [String] ID for an ActiveFedora::Base work with FileSets
+  # This zips up all the files in a FileSet object
```

Comment:
Instead of `# This zips up all the files in a FileSet object`, I think it should be `# This zips up all the files in a work object`